public/metrics/service.php

```diff
@@ -140,6 +140,7 @@ public function serveMetrics(): void
     {
         header('Content-Type', 'text/plain');
         echo $this->getServiceMetrics();
+        unset($this->guzzle_client);
```

Comment:
We tested this, but it had no visual effect. 
Do we still think it's needed, due to *good-housekeeping*, or should we consider removing?

Reply:
I don't think it can do any harm.